htdocs/lib/js/time-element.js

```diff
@@ -0,0 +1,616 @@
+(function() {
+  'use strict';
+
+  // Shout out to https://github.com/basecamp/local_time/blob/master/app/assets/javascripts/local_time.js.coffee
```

Comment:
I'd like to acknowledge the copyright a little more formally, e.g. a comment 
"This file derived from the Basecamp Local Time library
Copyright 2015 Javan Makhmali, Basecamp
MIT license"